packages/curve-ui-kit/src/shared/ui/Slider.tsx

```diff
@@ -0,0 +1,24 @@
+import { Stack } from '@mui/material'
+import MuiSlider, { type SliderProps as MuiSliderProps } from '@mui/material/Slider'
+export type SliderProps = MuiSliderProps
+
+/**
+ * Slider component built on top of MUI Slider component
+ * @param props - SliderProps
+ * @returns Slider component with custom rail background and flex wrapper to prevent height issues and overflows
+ */
+export const Slider = (props: SliderProps) => {
+  const { orientation = 'horizontal' } = props
+  const isHorizontal = orientation === 'horizontal'
```

Comment:
Code like this assumes `orientation` only has 2 options, it breaks when you want to add a 3rd option.
Generally it's more clear and type-safe to use mappings instead of ternaries